R/zchunk_LA1231.elec_tech.R

```diff
@@ -44,6 +44,15 @@ module_energy_LA1231.elec_tech<- function(command, ...) {
       filter(supplysector == "electricity")%>%
       distinct(supplysector, technology, minicam.energy.input) -> energy.GAS_TECH
 
+    # Obtain unique gas technologies to avoid hard coding technologies later in the code
+    # NOTE: It is assumed that there are 2 gas technologies only. If new gas technologies are added in the future, then this chunk has to be updated.
+    calibrated_techs %>%
+      filter(subsector == "gas")%>%
+      filter(supplysector == "electricity")%>%
+      distinct(technology) -> energy.GAS_TECH_CONST
+    energy.GAS_TECH_CONST$technology[1] -> GAS_TECH1
+    energy.GAS_TECH_CONST$technology[2] -> GAS_TECH2
+
```

Comment:
Can you put in a comment that Gas_Tech1 is gas steam/CT and Gas_Tech2 is gas CC.